tests/setup.ts

```diff
@@ -63,13 +63,24 @@ class FakeBroadcastChannel extends EventTarget {
 
 expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }) as any);
 
-// We want to detect jest sometimes
-(global as any).__JEST__ = (global as any).__JEST__ || {};
+// Don't serialize Timeout objects in depth
+// TODO(dsanders11): This feels like we shouldn't need to do this?
+expect.addSnapshotSerializer({
+  serialize(val) {
+    return val.toString();
+  },
+  test(val) {
+    return val?.constructor.name === 'Timeout';
+  },
+});
```

Comment:
Spun my wheels on this one for a bit, the `layoutDebounce` property in `EditorMosaic` was serializing to a full complex object, unlike a simple integer under Jest. The solution I came up with was to just use this custom serializer so that it serializes to `[object]` in the snapshot, which is fine for our needs.